packages/legend-query/src/stores/QueryBuilderExplorerState.ts

```diff
@@ -305,30 +335,35 @@ export const getPropertyNodeMappingData = (
         graphManagerState,
         parentTargetSetImpl,
       );
+      const propertyMappingsFromSuperSetImpls = superSetImplementations
+        .map((s) => resolvePropertyMappingsForSetImpl(graphManagerState, s))
+        .flat();
       const mappedProperties = propertyMappings
```

Comment:
can we have one array of mappedProperties concating superSetMappedProperties

Reply:
If one property is mapped both in the current class Mapping **(A)** and the class mapping **(B)** which it entends from, I think A takes precedence over B. Therefore I chose not to concatenate them. :)

Reply:
yeah but concat doesn't change this. you add the superset to the end. the find() grabs the first one. if its A it would grab A.